featurebyte/query_graph/node/base.py

```diff
@@ -926,6 +994,23 @@ def generate_odfv_expression(self, left_operand: str, right_operand: str) -> str
         """
         return self.generate_expression(left_operand, right_operand)
 
+    def generate_odf_expression(self, left_operand: str, right_operand: str) -> str:
```

Comment:
I wonder if it would be clearer if we name this as `generate_udf_expression()` since `udf` is the well known acronym and is actually accurate in this context?

And at the same time rename `derive_on_demand_function_code()` to  `derive_user_defined_function_code()` to distinguish it from feast's `derive_on_demand_view_code()`?

Reply:
We have also have `udf` in featurebyte...

Reply:
I see, in that case probably better to keep as is.

Reply:
Maybe can consider making it more specific like `databricks_udf` to make it clear it's not related to featurebyte udf. Or maybe a comment in the docstring is sufficient.

Reply:
Rename it to `udf` to make it clearer.